flow/probes/ovssflow.go

```diff
@@ -242,6 +243,7 @@ func (o *OvsSFlowProbesHandler) registerProbe(n *graph.Node, capture *types.Capt
 
 // RegisterProbe registers a probe on a graph node
 func (o *OvsSFlowProbesHandler) RegisterProbe(n *graph.Node, capture *types.Capture, e FlowProbeEventHandler) error {
+	o.Node = n
```

Comment:
it doesn't seem to be required. as it is passed as argument of registerProbe

Reply:
Done